### DISCUSSION TITLE
Getting Agent embedded@00 exited unexpectedly

### DISCUSSION THREAD

**Discussion by bhakarswayamavarapu** [Xceptance/XLT#396] [General]
Hi team,

Getting below response when I was trying run the load test in embedded mode.
**Agent embedded_00@local host exited unexpectedly with exit code(3).**

I have used the below command:
cd <binpath> mastercontroller.cmd -auto -embedded -comment "My project"

![MicrosoftTeams-image](https://github.com/Xceptance/XLT/assets/140174174/44ea0fab-bf2e-4fa8-94c0-1ec774057449)
["](url)

Can you please suggest? Thanks!

**Comment by jowerner** [comment]
This happened because your test scenario failed too often. An XLT agent terminates if a certain number of errors is reached. See the following setting in `config/default.properties`:

```
## The maximum number of errors allowed before an agent terminates.
## Useful in case of severe server errors, etc.
com.xceptance.xlt.maxErrors = 1000
```

There is likely an issue with your scenario. Please consult the agent log files in the folder results to get an idea what went wrong.
